cmd/nitro/nitro.go

```diff
@@ -122,6 +122,30 @@ func main() {
 	os.Exit(mainImpl())
 }
 
+// Runs metrics server at address:port specified by config if metrics flag is
+// enabled. Additionally if pprof at specified pprof port when it's enabled.
+func mustRunMetrics(cfg *NodeConfig) {
```

Comment:
Why is this function called mustRunMetrics?

Reply:
MustXYZ is a convention in Go [0]. When helper stops the program (usually called from main) convention is to name it with mustXYZ.

[0] https://google.github.io/styleguide/go/decisions#must-functions